zuul.d/ansible-utils-jobs.yaml

```diff
@@ -12,58 +12,34 @@
       ansible_collections_repo: github.com/ansible-collections/ansible.utils
 
 - job:
-    name: ansible-test-integration-ansible-utils-python27
+    name: network-ee-integration-tests-utils
     parent: ansible-test-integration-ansible-utils
```

Comment:
this is the wrong parent, we do not want to have 'ansible-test' jobs in the inheritance of the new job structure.

See: https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/vyos-vyos-jobs.yaml#L3